tests/catch/unit/device/hipDeviceSynchronize.cc

```diff
@@ -40,6 +43,37 @@ static __global__ void Iter(int* Ad, int num) {
     }
 }
 
+TEST_CASE("Unit_hipDeviceSynchronize_Positive_Empty_Streams") {
+  const auto device = GENERATE(range(0, HipTest::getDeviceCount()));
+  HIP_CHECK(hipSetDevice(device));
+  INFO("Current device: " << device);
+
+  hipStream_t stream;
+  HIP_CHECK(hipStreamCreate(&stream));
+  HIP_CHECK(hipDeviceSynchronize());
+  REQUIRE(true);
```

Comment:
Why this?

Reply:
Was mistakenly left in. Please proceed with the review.